Update documentation about testing (#324)
wpaccessibility/wp-a11y-docs: docs/topics/design/color/color-contrast-text.md

This requires that the contrast between background and foreground colors has a luminosity contrast ratio of:

- 4.5:1 for normal text.
- 3:1 for large text (24px equivalent or 19px equivalent and bold).

{: .callout  .tip }
**Tip:** Logos and logotype are exceptions to accessibility guidelines, including color contrast rules. If you’re designing a new logo, however, it might be a good idea to consider them, anyway.

There are many tools to check color contrast ratios. WebAxe published an overview of contrast checkers. You can test color contrast with these color checkers:

> - [contrastchecker.com](http://contrastchecker.com/) (Online) — checks foreground/background contrast levels and also gives information on the accessibility levels of the contrast

Reviewer: ```suggestion
- [contrastchecker.com](https://contrastchecker.com/) (Online) — checks foreground/background contrast levels and also gives information on the accessibility levels of the contrast
```

> - [Colour Contrast Analyser (CCA)](http://www.paciellogroup.com/resources/contrastAnalyser), Vispero’s free color contrast checker for Windows and Mac, includes text and visual elements for individuals with color-blindness or low vision impairments.

Reviewer: ```suggestion
- [Colour Contrast Analyser (CCA)](https://vispero.com/lp/color-contrast-checker/), Vispero’s free color contrast checker for Windows and Mac, includes text and visual elements for individuals with color-blindness or low vision impairments.
```

> - [Background Image & Text Contrast Checker](http://www.brandwood.com/a11y/), by Andrew Brandwood.

Reviewer: ```suggestion
- [Background Image & Text Contrast Checker](https://www.brandwood.com/a11y/), by Andrew Brandwood.
```

- [Color Contrast And Why You Should Rethink It](https://www.smashingmagazine.com/2014/10/color-contrast-tips-and-tools-for-accessibility/)

> - [WCAG Color Contrast Guideline](http://mcdpartners.com/lab/meeting-wcag-color-contrast-guideline/)

Reviewer: ```suggestion
- [WCAG Color Contrast Guideline](https://www.mcdpartners.com/news/meeting-wcag-color-contrast-guideline)
```

> - [Color Contrast and why you should rethink it](https://www.smashingmagazine.com/2014/10/color-contrast-tips-and-tools-for-accessibility/)

Reviewer: Duplicate from above; delete

Author: deleted

> - [Sim Daltonism](http://michelf.ca/projects/sim-daltonism/) — a color-blindness simulator for Mac OS X.

Reviewer: ```suggestion
- [Sim Daltonism](https://michelf.ca/projects/sim-daltonism/) — a color-blindness simulator for Mac OS X.
```
